### ISSUE TITLE
Doesn't work with hash/chunking

### ISSUE THREAD

**Issue by StevenWeathers** [bengsfort/rollup-plugin-generate-html-template#7]
Rollup allows hashing/chunking, your plugin only works with the singular static bundle file.
Would be nice if it worked with both solutions from Rollup.

**Comment by bengsfort** [comment]
Can you elaborate? Are you referring to multiple outputting multiple entry that are not dynamically loaded?

**Comment by StevenWeathers** [comment]
If you use rollups options like below, it throws errors:
```
{
dest: "somedir",
entryFileNames: "[name]-[hash].js",
chunkFileNames: "[name]-[hash].js"
}
```

**Comment by bengsfort** [comment]
Great, thanks! Will check this out tomorrow.

**Comment by bengsfort** [comment]
This should be fixed with `v1.2.0`, which should now be published to both NPM and Yarn.
